common-libs/internal-api-clients/src/helpers/DidAuthAdapter.ts

```diff
@@ -17,6 +18,7 @@ export class DidAuthAdapter {
   }
 
   isTokenExpired(token: string): boolean {
-    return this._didAuthService.isTokenExpired(token)
+    const LocalExpiringDidAuthResponseTokenCheck = LocalExpiringDidAuthResponseToken.initialize(this.tokenRequestTime, token)
```

Comment:
We only use names starting with capital letters for classes and types (and constants which are SNAKE_CASE).

`npm run lint` should have complained about this.

Just rename it to `const localExpiringDidAuthResponseTokenCheck`